evaluation/benchmarks/swe_bench/run_infer.py

```diff
@@ -108,7 +109,9 @@ def get_instruction(instance: pd.Series, metadata: EvalMetadata) -> MessageActio
     llm_model = metadata.llm_config.model
 
     # Determine the template file based on mode and LLM
-    if mode.startswith('swt'):
+    if INSTRUCTION_TEMPLATE_NAME:
+        template_name = INSTRUCTION_TEMPLATE_NAME
```

Comment:
actually another quetsion: you might want to log this to eval metadata for tracking purpose

Reply:
Makes sense. I've put it in EvalMetadata now.